### PR TITLE
Deploy updates

### PR DESCRIPTION
**Pull request by ultimatemonty**
Minor updates to the config files for deployment. also updates the README with light deets on deployment

/cc @pdougall1 @gschorkopf @camthompson

### CODE REVIEW

File: config/deploy.js [hunk 1]
```diff
@@ -18,19 +18,18 @@ module.exports = function(deployTarget) {
...
   }
 
-  // TODO: need to configure a production env that is not scoped to atl-ember-staging
-  // if (deployTarget === 'production') {
-  //   ENV.build.environment = 'production';
-  //   ENV.s3.bucket = "atl-ember-staging";
-  //   ENV.s3.region = "us-west-2";
-  //   ENV.cloudfront.distribution = process.env.PRODUCTION_DISTRIBUTION;
-  // }
+  if (deployTarget === 'production') {
```
Comment (gschorkopf) [inline config/deploy.js]: Can lines 21 through 31 be refactored as something like:

```js
ENV.build.environment = deployTarget;
ENV.s3.bucket = process.env.S3_BUCKET;
ENV.s3.region = process.env.S3_REGION;
ENV.cloudfront.distribution = process.env.CF_DISTRIBUTION;

Comment (gschorkopf) [inline config/deploy.js]: Cut down on those conditionals?

**Comment by gschorkopf** [comment]
Consider my comments, then :ship:

**Comment by ultimatemonty** [comment]
@gschorkopf my feeling is it shouldn't be a pre-requisite to need the rails API running in order to work on the client. Mirage will take care of mocking the endpoint requests. We could possibly add another environment like `development-api` or `local-api` that did this if desired. Or the user could just change the config if needed?

**Comment by gschorkopf** [comment]
I mean yeah, the user could also just use a flag on `ember s` as well, I just thought that defaulting to asking the port for responses would make sense. We don't have mirage set up, right?

**Comment by pdougall1** [comment]
It sounds like it's pretty important to get mock data backing the app pretty soon, so we can remove that dependancy.  As it stands there is probably not much that the app can do without Rails, so it might make sense to keep it for the time being.

**Comment by ultimatemonty** [comment]
Mirage is setup and has a basic endpoint (empty response) for events.
Nothing else is setup atm but could do it pretty easily.

**Comment by pdougall1** [comment]
@ultimatemonty agreed, this looks great!  I'll go ahead and merge it.  I'll also try to get another PR filling out those other models with some Mirage data so we get rid of that Rails dependency entirely, because that was a good point.  I guess I'll also make an issue, to help remind me.

**Pull request merged**